api/python/provisioner/scm/saltstack/rc_sls/consul.py

```diff
@@ -77,13 +75,13 @@ class ConsulConfigSLS(ConsulSLS):
     def setup_roots(self):
         super().setup_roots()
 
-        config = {}
-        pillar = {'config': config, 'service': self.state.service}
+        consul_config = {}
```

Comment:
I would recommend keeping the dictionary name as config (as we are nywaz within the context of consul.py)